### PR TITLE
Make the getstats API aligned with the cache screen 

### PR DESCRIPTION
**Pull request by eugene-eko2000**
See the screenshot to understand the change

Simulator Screenshot - iPhone 15 Pro - 2023-12-28 at 01 01 21

### CODE REVIEW

File: be/db_test.go [hunk 1]
```diff
@@ -497,6 +497,67 @@ func TestGetTopScores(t *testing.T) {
...
+		recordColumns := []string{
+			"kitns_daily",
+			"kitns_disbursed",
+			"kitns_ref_daily",
+			"kitns_ref_disbursed",
+		}
+		for _, testCase := range testCases {
+			setUp()
+			if testCase.expectError {
+				mock.ExpectQuery("SELECT kitns_daily, kitns_disbursed, kitns_ref_daily, kitns_ref_disbursed	FROM users WHERE id = (.+)").
```
Comment (eldarm) [inline be/db_test.go:540]: May be format it for readability with ``?

File: be/db_test.go [hunk 2]
```diff
@@ -497,6 +497,67 @@ func TestGetTopScores(t *testing.T) {
...
+					WithArgs(testCase.id).
+					WillReturnError(fmt.Errorf("error getting kitns"))
+			} else {
+				mock.ExpectQuery("SELECT kitns_daily, kitns_disbursed, kitns_ref_daily, kitns_ref_disbursed	FROM users WHERE id = (.+)").
```
Comment (eldarm) [inline be/db_test.go:544]: Ditto.

**Review by eldarm — APPROVED** [review]

File: be/db_test.go (hunk 1 above)
Comment (eugene-eko2000) [reply be/db_test.go:540]: It matches the regexp. So even a number of spaces might be critical there. When it's in one line it's easier to count them and tweak it if necessary.

File: be/db_test.go (hunk 2 above)
Comment (eugene-eko2000) [reply be/db_test.go:544]: See the commant above.

**Pull request merged**